front/app/containers/Admin/pagesAndMenu/containers/Attachments/index.tsx

```diff
@@ -1,28 +1,28 @@
 import React from 'react';
 
 // components
-import SectionFormWrapper from '../../components/SectionFormWrapper';
 import {
```

Comment:
Ignore the import sections of all files. It's my auto-format tool that rearranges them. We could look into doing the whole codebase in one PR, although I remember it wasn't straightforward with the tool I'm using.

Reply:
The changes look fine but I'd suggest disabling that for PRs in the future and seeing if formatting the whole codebase at once is possible. Not sure doing it file-by-file is worth the bigger diff or needing to ignore entire aspects of a PR